src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -179,6 +179,21 @@ const userUpdate = (userID: string, userPrivateKey?: PrivateKey<Uint8Array>, sta
     errorCode: ErrorCodes.USER_UPDATE_REQUEST_FAILURE,
 });
 
+const userKeyUpdateApi = (userID: string, userPrivateKey: PrivateKey<Uint8Array>, augmentationFactor: AugmentationFactor, userKeyId: number): RequestMeta => ({
```

Comment:
We should add a docblock above this method about what it does. You don't need to do any of the `@param` stuff but a sentence or two about what this is doing would be good.